cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxEditBoxHelper.java

```diff
@@ -172,6 +182,7 @@ public void run() {
                             Log.d(TAG, "edit box get focus");
                         } else {
                             editBox.setVisibility(View.GONE);
+                            mHandler.postDelayed(decor_view_settings, 500);
```

Comment:
i'm not sure should you remove immediate hide, because it works on most android's and this fix is needed only on some devices. But it safe to call hideVirtualButton again